Source/Core/VideoCommon/VertexLoader_Color.h

```diff
@@ -4,14 +4,42 @@
 #pragma once
 
 #include "Common/CommonTypes.h"
+#include "Common/EnumMap.h"
+#include "VideoCommon/CPMemory.h"
 #include "VideoCommon/VertexLoader.h"
 
-enum class VertexComponentFormat;
-enum class ColorFormat;
-
 class VertexLoader_Color
 {
+  template <typename T, auto last_member>
+  using EnumMap = typename Common::EnumMap<T, last_member>;
+
+  using SizeTable = EnumMap<EnumMap<u32, ColorFormat::RGBA8888>, VertexComponentFormat::Index16>;
+
 public:
-  static u32 GetSize(VertexComponentFormat type, ColorFormat format);
+  static DOLPHIN_FORCE_INLINE u32 GetSize(VertexComponentFormat type, ColorFormat format)
```

Comment:
`DOLPHIN_FORCE_INLINE` is probably a good idea for here, but you should include `Common/Inline.h` if you're using it.

Reply:
`DOLPHIN_FORCE_INLINE` seemed to slightly increase performance for me. It's up to ~118 FPS now.